modules/apps/client-extension/client-extension-web/test/js/AttribureFields.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {fireEvent, render} from '@testing-library/react';
```

Comment:
Please change the file type to `*test.tsx`. We want to prefer typescript whenever possible.